tests/integration/test_fim/test_synchronization/test_response_timeout.py

```diff
@@ -154,7 +154,7 @@ def update_agent_datetime():
     check_apply_test({'response_timeout'}, get_configuration['tags'])
 
     # Variables
-    LOG_PATH = "/var/ossec/logs/ossec.log"
+    LOG_PATH = "/var/ossec/logs/wazuh.log"
```

Comment:
There's already a variable for this, so there's no point to define a new one. We can fix this by using the variable `LOG_FILE_PATH`

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/commit/20fb78d2a4218a84aa5ce2a4323ff3437686b23b